transport/transporters/rabbitmq/transporter/connection.go

```diff
@@ -0,0 +1,108 @@
+/*
+  Copyright 2019 Nextdoor.com, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package transporter
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/NeowayLabs/wabbit"
+	"github.com/cenkalti/backoff"
+	"github.com/sirupsen/logrus"
+)
+
+//go:generate mockgen -destination=mock_transporter.go -package=mocks github.com/Nextdoor/pg-bifrost.git/transport/transporters/rabbitmq/transporter ConnectionGetter
```

Comment:
Need to change destination to `mocks/mock_transporter.go` otherwise it dumps it into this same directory. Also please move this to `transporter.go`.